intermittent lint failure

## Observed behavior

see https://github.com/learningequality/kolibri/pull/8500#pullrequestreview-813868553

It appears that some invalid code slipped in without triggering a lint failure. At a later point, this caused a lint failure in my personal fork during unrelated work.

## Errors and logs

This block:

https://github.com/learningequality/kolibri/blob/80fe54f6ace6ed3146099984a928c11e58c8bc0a/kolibri/utils/server.py#L925-L929

triggered this linting error in my personal branch:


```
kolibri/utils/server.py:929:40: F812 list comprehension redefines 'output' from line 927
```


## Expected behavior

* The linter should have flagged this in the PR
* ~The code in question should not redefine `output`~ fixed in #8773 

## User-facing consequences

Possible that bugs would slip through

## Steps to reproduce

uncertain


## Context

0.15.0-beta2

--- a/kolibri/utils/server.py
+++ b/kolibri/utils/server.py
@@ -926,7 +926,7 @@ def get_installer_version(installer_type):  # noqa: C901
             if hasattr(output, "decode"):  # needed in python 2.x
                 output = output.decode("utf-8")
             package_info = output.split("\n")
-            version_info = [output for output in package_info if "Version" in output]
+            version_info = [line for line in package_info if "Version" in line]
             if version_info:
                 version = version_info[0].split(":")[1].strip()
                 return version

--- a/kolibri/utils/tests/test_server.py
+++ b/kolibri/utils/tests/test_server.py
@@ -46,6 +46,25 @@ class TestServerInstallation(object):
             ].format("1.0")
 
     @mock.patch("sys.argv", ["/usr/bin/kolibri", "start"])
+    def test_dpkg_version(self):
+        DPKG_OUTPUT = """
+        Package: kolibri
+        Status: install ok installed
+        Priority: optional
+        Section: education
+        Architecture: all
+        Source: kolibri-source
+        Version: 0.15.0~beta2-0ubuntu1
+        Depends: python3 (>= 3.4), python3-pkg-resources, adduser
+        Recommends: python3-cryptography (>= 1.2.3)
+        """
+        with mock.patch("kolibri.utils.server.check_output", return_value=DPKG_OUTPUT):
+            install_type = server.installation_type()
+            assert install_type == installation_types.install_type_map[
+                installation_types.DEB
+            ].format("0.15.0~beta2-0ubuntu1")
+
+    @mock.patch("sys.argv", ["/usr/bin/kolibri", "start"])
     @mock.patch("os.environ", {"KOLIBRI_INSTALLER_VERSION": "1.0"})
     def test_apt(apt):
         with mock.patch("kolibri.utils.server.check_output", return_value="any repo"):

